backend/src/apiserver/server/api_converter.go

```diff
@@ -1348,11 +1366,14 @@ func toModelRun(r interface{}) (*model.Run, error) {
 			},
 		},
 		RunDetails: model.RunDetails{
+			State:                   state.ToV2(),
```

Comment:
Not to be addressed now, but another incidence of the `ToV2()` vs. `ToV1()` vs. "nocall" confusion.